automatic/kubernetes-kompose/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,28 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName = 'kubernetes-kompose'
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$installLocation = "$(Get-ToolsLocation)\$packageName"
+
+$packageArgs = @{
+  PackageName    = $packageName
+  FileFullPath64 = Get-Item $toolsPath\kompose*.exe.tar.gz
+  Destination    = $installLocation
+}
+Get-ChocolateyUnzip @packageArgs
+
+if (!(Test-Path $installLocation -include "kompose*.exe")) {
```

Comment:
This test would also match `kompose.exe`, I believe.
So this test won't work on upgrade.